modules/ublog/src/main/UblogBlog.scala

```diff
@@ -1,23 +1,47 @@
 package lila.ublog
 
-import lila.core.perf.UserWithPerfs
 import lila.core.perm.Granter
 
 case class UblogBlog(
     _id: UblogBlog.Id,
-    tier: UblogRank.Tier,           // actual tier, auto or set by a mod
-    modTier: Option[UblogRank.Tier] // tier set by a mod
+    tier: UblogBlog.Tier,           // actual tier, auto or set by a mod
+    modTier: Option[UblogBlog.Tier] // tier set by a mod
 ):
   inline def id = _id
-  def visible   = tier >= UblogRank.Tier.UNLISTED
-  def listed    = tier >= UblogRank.Tier.LOW
+  def visible   = tier > UblogBlog.Tier.HIDDEN
+  def listed    = tier > UblogBlog.Tier.HIDDEN
```

Comment:
what is Tier.UNLISTED about now, if `Tier.UNLISTED.listed == true`?

Reply:
don't quite remember that looks like an oversight when i was tier flattening. there is a functional difference between them at present - unlisted can still be seen on the user's blog page whereas hidden cannot. we should preserve that and roll those two lines back.